test/radio/radio-controller-test.ts

```diff
@@ -1,11 +1,14 @@
+import * as net from 'net';
 import test from 'ava';
 import * as dgram from 'dgram';
 import {RadioController, UDPScanConfig} from '../../src/radio/radio-controller';
-import {UdpResponse} from '../../src';
+import {UdpResponse, TcpResponse} from '../../src';
 
 const UDP_DUMMY_BUFFER_1 = Buffer.from('test UDP buffer');
 const UDP_DUMMY_BUFFER_2 = Buffer.from('foo bar lorem ipsum');
 
+const TCP_DUMMY_BUFFER = Buffer.from('dummy data');
```

Comment:
Let's change this to 'placeholder data'.

Reply:
And call the var `TCP_PLACEHOLDER_BUFFER`.

Reply:
Done